config/tag-processor-test.yaml

```
@@ -0,0 +1,45 @@
+        code: |
+          console.log(JSON.stringify(msg.meta, null, 2));
+          console.log(JSON.stringify(msg.payload, null, 2));
+          msg.meta.mqtt_topic = msg.meta.topic.replace(/\./g, "/")
+          return msg;
+
+output:
+  #stdout: {}
+  mqtt:
+    urls: ["tcp://10.13.37.202:1883"]
+    client_id: 'benthos-umh-test'
+    topic: '${! meta("mqtt_topic") }'
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)